library/packages/src/lib/packages/repository_product.rb

```diff
@@ -0,0 +1,17 @@
+module Packages
+  class RepositoryProduct
```

Comment:
I think we should define a base class for sharing the common resolvable data, see https://github.com/yast/yast-pkg-bindings/blob/d91b7dd06db0b7a316ef0ea2506abdba4664cdfb/src/Resolvable_Properties.cc#L187. At the beginning it adds the generic data defined for all resolvables, then the object class specific data.